### PR TITLE
Use more verbal `isCoolBackground` in noun store

### PR DESCRIPTION
**Pull request by nikitattt**
Use `isCoolBackground` in `store/noun`. It's more nounish wording, so it's easier to understand what it means as both nouns' backgrounds are 'grey'.

### CODE REVIEW

File: packages/webapp/package.json [hunk 1]
```diff
@@ -30,7 +30,6 @@
     "dayjs": "^1.10.7",
     "ethereum-bloom-filters": "^1.0.10",
     "ethers": "^5.5.1",
-    "history": "^5.0.1",
```
Comment (lastpunk9999) [inline packages/webapp/package.json:33]: Was this accidental inclusion or necessary for this PR?

File: packages/webapp/package.json [hunk 2]
```diff
@@ -71,5 +70,8 @@
   "devDependencies": {
     "@types/websocket": "^1.0.4",
     "dotenv": "^10.0.0"
+  },
+  "resolutions": {
```
Comment (lastpunk9999) [inline packages/webapp/package.json:74]: Was this accidental inclusion or necessary for this PR?

File: packages/webapp/package.json (hunk 1 above)
Comment (nikitattt) [reply packages/webapp/package.json:33]: Netlify build results in error without this due to dependencies mixup problems. The issue is only observed on Netlify, builds locally without a problem.

**Review by lastpunk9999 — APPROVED** [review]
Thanks for your quick replies. All looks good!

**Pull request merged**